examples/nuclio/query/query_example.go

```diff
@@ -46,30 +47,33 @@ func Handler(context *nuclio.Context, event nuclio.Event) (interface{}, error) {
 		return nil, err
 	}
 
-	// convert string times (unix or RFC3339 or relative like now-2h) to unix milisec times
+	// Convert time strings to Unix timestamp in milliseconds integers.
+	// The input time string can be of the format "now", "now-[0-9]+[mdh]"
+	// (for example, "now-2h"), "<Unix timestamp in milliseconds>", or
+	// "<RFC3339 time>" (for example, "2018-09-26T14:10:20Z").
 	from, to, step, err := utils.GetTimeFromRange(query.From, query.To, query.Last, query.Step)
 	if err != nil {
-		return nil, errors.Wrap(err, "Error parsing query time range")
+		return nil, errors.Wrap(err, "Error parsing query time range.")
 	}
 
-	// Create TSDB Querier
+	// Create a TSDB Querier
 	context.Logger.DebugWith("Query", "params", query)
 	adapter := context.UserData.(*tsdb.V3ioAdapter)
 	qry, err := adapter.Querier(nil, from, to)
 	if err != nil {
 		return nil, errors.Wrap(err, "Failed to initialize Querier")
 	}
 
-	// Select Query to get back a series set iterator
+	// Select a query to get back a series-set iterator
 	set, err := qry.Select(query.Name, strings.Join(query.Aggregators, ","), step, query.Filter)
 	if err != nil {
 		return nil, errors.Wrap(err, "Select Failed")
 	}
 
-	// convert SeriesSet to Json (Grafana simpleJson format)
+	// Convert a SeriesSet to a JSON object (Grafana simpleJson format)
 	f, err := formatter.NewFormatter("json", nil)
 	if err != nil {
-		return nil, errors.Wrap(err, "failed to start json formatter")
+		return nil, errors.Wrap(err, "Failed to start the JSON formatter.")
```

Comment:
Note, we usually tend to start error messages with lower case because of possibility of chaining multiple error messages into one string

Reply:
I changed all messages that seem to be standalone sentences to complete sentences that begin with a capital letter and usually end in a period. I think the overwhelming majority, if not all, of these messages are not concatenated into one sentence; (it's OK if they're concatenated into one message with multiple sentences). It looks back when you have a log message that's an actual sentence but begins with a lower-case letter.

Reply:
UPDATE: As discussed f2f, Eran D. says that log messages should be capitalized and while Go has a convention of beginning error messages with a lower-case letter, our convention is to capitalize error messages as well ==> my changes are consistent with our internal conventions. I made the changes in all (or almost all) non-test files.